draft-ietf-tls-tls13.md

```diff
@@ -3356,7 +3384,7 @@ Based on these inputs, the messages then contain:
 Certificate
 : The certificate to be used for authentication, and any
   supporting certificates in the chain. Note that certificate-based
-  client authentication is not available in 0-RTT mode.
+  client authentication is not available in resumption (or 0-RTT) flows.
```

Comment:
Please double-check this, but the pictures don't show a server Certificate for resumption.

Reply:
Yes.  This is correct.  "PSK" rather than resumption perhaps?